decidim-core/app/presenters/decidim/user_presenter.rb

```diff
@@ -33,13 +35,14 @@ def avatar
     end
 
     def avatar_url(variant = nil)
+      return default_avatar_url if __getobj__.blocked?
       return avatar.default_url unless avatar.attached?
 
       avatar.path(variant:)
     end
 
     def default_avatar_url
-      attached_uploader.default_url
+      attached_uploader(:avatar).default_url
```

Comment:
Just a note to myself that I have another PR (#9596) with the same fix but I think we can still merge this because it'll take some time to get my PRs reviewed. I can just then resolve the conflict at #9596.

I understand it is needed with this PR because the method is right now broken.